src/products/filters/models/ProductsFilterApiResponse.ts

```diff
@@ -0,0 +1,16 @@
+import AlphabeticCurrencyCode from '../../../model/AlphabeticCurrencyCode';
+import ChannelApeApiError from '../../../model/ChannelApeApiError';
+
+export default interface ProductsFilterApiResponse {
+  alphabeticCurrencyCode: AlphabeticCurrencyCode;
```

Comment:
This doesn't look needed either.

Reply:
I was following the same pattern as the variant model, which has VariantApiResponse and Variant.  It is mainly needed because the incoming JSON response has the dates as strings, but the SDK needs to have them returned as dates.

Reply:
@mb-ca just meant it looks like ProductFilterApiResponse.ts already exist. Just looked like a duplicate of that. I understand why you need a separate response object.